### PR TITLE
Upgrading litmusportal charts to Litmus-2.0.0-Beta3

### PR DESCRIPTION
**Pull request by rajdas98**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] DCO signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md

### CODE REVIEW

**Review by Jasstkn — COMMENTED** [review]
You changes aren't compatible with earlier 1.16 / 1.17 k8s versions.

**Review by Jasstkn — COMMENTED** [review]
And also:
> ✖︎ litmus-2-0-0-beta => (version: "2.0.0-Beta3", path: "charts/litmus-2-0-0-beta") > Chart version not ok. Needs a version bump!

File: charts/litmus-2-0-0-beta/templates/ingress.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            name: litmusportal-frontend-service
+            port:
+              number: 9091
+      - path: /backend/(.*)
+        backend:
+          service:
+            name: litmusportal-server-service
+            port:
+              number: 9002
+    {{- else if semverCompare "<=1.19-0" .Capabilities.KubeVersion.GitVersion }}
```
Comment (Jasstkn) [inline charts/litmus-2-0-0-beta/templates/ingress.yaml:52]: remove = sign

Comment (rajdas98) [reply charts/litmus-2-0-0-beta/templates/ingress.yaml]: Done

**Review by Jasstkn — APPROVED** [review]

**Comment by rajdas98** [comment]
@Jasstkn I have push a new commit which is disabling the ingress

**Review by Jasstkn — APPROVED** [review]

**Review by ksatchit — APPROVED** [review]

**Pull request merged**